README.md

```diff
@@ -239,7 +239,7 @@ Below is the schema of the performance measures that are logged to the console.
 
 ```js
 {
-  component, // Name of the component
+  componentName, // Name of the component
```

Comment:
the variable clearly states that this is a component name :)
we can remove the comment